draft-ietf-rtcweb-jsep.xml

```diff
@@ -2409,13 +2409,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
             <t>The m= line and corresponding "a=rtpmap" and "a=fmtp"
             lines MUST only include codecs present in the most recent
-            answer which have not been excluded by the codec preferences
-            of the associated transceiver. Note that non-JSEP endpoints
-            are not subject to these restrictions, and might offer media
-            formats that were not present in the most recent answer, as
-            specified in <xref target="RFC3264"></xref>, Section 8.
-            Therefore, JSEP endpoints MUST be prepared to receive such
-            offers.</t>
+            answer which have not been excluded by the codec
+            preferences of the associated transceiver.  However, if
+            the offer was created with the "ReofferCodecs" option
+            (<xref target="sec.reoffer-codecs"></xref>), these lines
+            SHOULD include all codecs that would be offered in an
+            initial offer, as defined in <xref
```

Comment:
Not a big fan of absolute statements followed by retractions. How about
"Unless the "ReoffereCodecs option was specified, the m=..."